Core/NoOpRunner.Core/Iterators/LatestOrderIterator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NoOpRunner.Core.Iterators
+{
+    class LatestOrderIterator : Iterator
```

Comment:
Why `LatestOrder`? What's that supposed to mean?

Reply:
Its like stack (Last in first out):
Data -> 1 2 3 4 5
LatestOrderIterator -> 5 4 3 2 1

Reply:
I'd say `BackwardIterator` or `ReverseIterator` would be more fitting in that case, but the problem is that it's only a `BackwardIterator` if you pass the appropriate param during construction, otherwise it's a `ForwardIterator`. And I still don't see how `LatestOrder` conveys how the iterator works

Reply:
Changed to `BackwardIterator` for now